index.js

```diff
@@ -27,7 +27,8 @@ const pLimit = concurrency => {
 
 		try {
 			await result;
-		} catch {}
+		// eslint-disable-next-line no-unused-vars
```

Comment:
@sindresorhus I added this `eslint-disable-next-line no-unused-vars` line or should anything be done with the `error` that is being catched?

(`no-unused-vars` rule allows `_` as var name, but your unicorn rules disallow using anything other than `error` to be used here)